### ISSUE TITLE
Last Watched into menu/folder

### ISSUE THREAD

**Issue by Misl3d** [Goldenfreddy0703/Otaku#82]
Feature Request/ Enhancement to turn the Last Watched item into a folder/menu that will list all the recently watched items not just 1 item.

**Comment by Goldenfreddy0703** [comment]
That's kinda literally what next up is for but good idea and will think about this more.

**Comment by Misl3d** [comment]
My bad, I honestly didn't even know that. Although I did play around with it, changing the sort method for Anilist and even with last updated sorting it wouldn't actually sort correctly. Like the last watched item and my Anilist activity show as Black Clover but in Next Up its the 10th item. Not sure if this is something with my Kodi config or just weirdness happening. I'm on Kodi v20. 

Thank you!!

**Comment by Goldenfreddy0703** [comment]
Actually watchlist can be weird sometimes to maybe I can make a history menu item if possible.

**Comment by DaisyF8** [comment]
+1 for History. I sometimes watch a 1 or 2 minutes of episodes that I want to pick up a day later. Would be good to have a place to find them.

**Comment by Goldenfreddy0703** [comment]
Hey so me and my team have been discussing this for awhile and we came to the conclusion that we don't need a History menu. Big reason why is cause we already have a History menu in watchlist called current which basically shows the anime you were watching. 

Here are a few steps for you guys on how to have your current menu to show your anime history in your watchlist. 

1. Go to Addon Settings and go to the Watchlist Category 
2. Make sure your Update Watchlist is on
3. Do these settings if you are Anilist, Kitsu, or Mal

- If you are using Anilist, change your Sort By to "Last Updated"
- If your are using Kitsu, change your Sort By to "Date Updated"
- If you are using Mal, change your Sort By to "Last Updated"

We have made a release today making these settings as a default for everyone that wants to access there current history. 

If you have anymore question on how to do this, feel free to ask.

**Comment by Misl3d** [comment]
That's exactly what I was looking for.. After reading this, I realized all my settings were correct but Current didn't match my AniList. 

After comparing Otaku Current to my AniList Watching, I saw that in Current the list was correct and sorted by Last Updated but only after several anime at the start of the List. Turns out the several anime at the start of the list were bugged/duplicates of anime further in the list and they were just stuck there even after watching another anime. 

So, I logged out and reauthorized AniList on Otaku and it corrected itself. So I guess something bugged out but everything works perfectly now. Thank you!
